src/utilities/awala/InternetEndpoint.ts

```diff
@@ -70,6 +75,33 @@ export class InternetEndpoint extends Endpoint {
     super(id, identityKeyPair, keyStores, {});
   }
 
+  public override async savePrivateEndpointChannel(
+    connectionParams: PrivateEndpointConnParams,
+    dbConnection?: Connection,
```

Comment:
could not override the method without making dbConnection optional

Reply:
Hmm, good point. In that case, we should probably use a different name, and tailor it to this particular app. How about `savePeerChannel()`?

"Private endpoint" makes sense in the core library, but in our case we can/should use "peer".